src/content/docs/guides/json/2-json_writing.mdx

```diff
@@ -82,6 +121,15 @@ Now that we have the `new_game_data` object that stores the same values as the J
   ```
   
   </TabItem>
+
+  <TabItem label="C#">
+
+  ```csharp
+  newGameData.ToFile("new_game_data.json");
```

Comment:
Function does not exist. See below:

`SplashKit.JsonToFile(newGameData, "new_game_data.json");`